src/pages/UserAdmin/index.tsx

```diff
@@ -59,7 +59,11 @@ const UserAdmin = () => {
       filters.user_study = userStudyChoice;
     }
     if (userClassChoice !== 0) {
-      filters.user_class = userClassChoice;
+      if (userClassChoice === 6) {
+        filters.user_class = -1;
+      } else {
+        filters.user_class = userClassChoice;
+      }
```

Comment:
Dette kan skrives på en enda litt mer elegant måte ved bruk av [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)-operatoren ;)

```suggestion
      filters.user_class = userClassChoice === 6 ? -1 : userClassChoice;
```